hms-app/src/components/RelevantIdeasLoader.tsx

```diff
@@ -4,30 +4,30 @@ import { Hackathon, Idea } from '../common/types'
 import { getIdeaDetails } from '../actions/IdeaActions'
 
 type Props = {
-  selectedHackweek: string
+  selectedHackathonID: string
```

Comment:
If we generally follow camelcase, we should change to it all the fields with `Id` in the suffix
```suggestion
  selectedHackathonId: string
```